acceptance/acceptance_test.go

```diff
@@ -1482,7 +1482,7 @@ func testAcceptance(
 								"build", repoName,
 								"-p", filepath.Join("testdata", "mock_app"),
 								"--env", "DETECT_ENV_BUILDPACK=true",
-								"--env", `ENV1_CONTENTS="Env1 Layer Contents From Command Line"`,
+								"--env", `"ENV1_CONTENTS=Env1 Layer Contents From Command Line"`,
```

Comment:
I'm a little concerned by this new change. I can't see this as expected behaviour from a user's perspective. What's the purpose of quoting the key and value?

I would have expected the following to work.

```
--env 'KEY_1=VAL1,KEY_2="VAL 2"'
```

|key|value|
|--- |--- |
| KEY_1 | VAL1 |
| KEY_2 | VAL 2 |

Reply:
Ok, I have tried implementing that, but this is way more complicated then I thought it will be - library does not support it so we have to parse it on our own.